modules/apps/object/object-test/src/testFunctional/paths/ObjectPortlet.path

```diff
@@ -72,7 +72,7 @@
 	</tr>
 	<tr>
 		<td>ENTRY_KEBAB_MENU</td>
-		<td>//div[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entry}')]/parent::div/following-sibling::div//button | //div[contains(@class,'dnd-tbody')]//*[contains(text(),'${key_entry}')]/following-sibling::div//button</td>
+		<td>//div[contains(@class,'dnd-tbody')]//*[contains(text(),'')]/parent::div/following-sibling::div//button</td>
```

Comment:
Hey @Italo-gabriel this path is used in some macros that are used in many tests, did you make a check if the change doesn't affect the tests?

I think that you forget to pass the `key_entry` variable in your modification, see this path: `//*[contains(text(),'')]` there are no `key_entry` variable. This can break many other tests that use the value of this variable.

cc: @timpak

Reply:
hey @evanilson-santana vanilson I'm going to return the PR and make the changes and confirm the operation again of all test cases that are using this PATH, I already send the changes